### PR TITLE
Decreasing learning rate, to avoid divergence.

### PR DESCRIPTION
**Pull request by dremovd**
Addressing issue #4

### CODE REVIEW

**Comment by lukedeo** [comment]
Hmm, seems reasonable - I'm going to push a few changes to this pretty soon (i.e., actual transposed convs, etc.) but seems harmless enough. What was your hardware setup that led to black images for the original learning rate as suggested in the DCGAN paper?

**Comment by dremovd** [comment]
I used gtx 1080 with 357 driver version. Back end was tensorflow, however, I think I tried Theano as well.

**Comment by minkooseo** [comment]
Observed the same black generated images when running on GeForce GTX 1080 with 367.57 driver. It was so with both of TensorFlow (0.12.1) and Theano (0.9.0b1) backends.
